frontend/public/components/edit-yaml.jsx

```diff
@@ -122,9 +122,6 @@ export const EditYAML_ = connect(stateToProps)(
         this.setState({ stale });
         if (nextProps.error) {
           this.handleError(nextProps.error);
-        } else if (this.state.error) {
-          //clear stale error state
-          this.setState({ error: '' });
```

Comment:
Do errors still get cleared when you click Reload? We'd want to clear errors in that case.